extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -166,6 +169,27 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyala-player'], fu
 				}
 			});
 
+			player.mb.subscribe( OO.EVENTS.PLAYHEAD_TIME_CHANGED, 'featured-video', function(eventName, time, totalTime) {
+				var secondsPlayed = Math.floor(time),
+					percentage = Math.round(time / totalTime * 100);
+
+				if ( secondsPlayed % 5 === 0 && secondsPlayed !== playTime ) {
```

Comment:
Under development the event occurred 3 to 5 times per second, that's why it had to be throttled.